validation-jsonast/js/src/main/scala/Ast.scala

```diff
@@ -0,0 +1,32 @@
+package jto.validation
+package jsonast
+
+import scala.scalajs.js
+import scala.scalajs.js.JSConverters._
+
+object Ast {
+  // https://www.scala-js.org/doc/interoperability/types.html
+  def to(ast: JValue): js.Dynamic =
+    (ast match {
+      case JString(value) => value
+      case JBoolean(value) => value
+      case JNull => null
+      case JArray(value) => value.map(to).toJSArray
+      case JObject(value) => value.mapValues(to).toJSDictionary
+      case JNumber(value) =>
+        val d = value.toDouble; if (d.isNaN || d.isInfinity) null else d
+    }).asInstanceOf[js.Dynamic]
```

Comment:
Equivalent code in circe, which goes through an `input: Any`: https://github.com/travisbrown/circe/blob/master/scalajs/src/main/scala/io/circe/scalajs/package.scala

I agree this could be reworked to look a bit nicer